From ce905b2c9fd432428bc96097eb4a99a237712eb3 Mon Sep 17 00:00:00 2001
From: Johannes Thoma <johannes@johannesthoma.com>
Date: Mon, 17 May 2021 19:04:35 +0200
Subject: [PATCH] rs_last_events of hd_struct always 0.

Sync throtteling does not work at the moment. We'll implement it
upon customer request. New patch that works with DRBD 9.0.29
---
 drbd/drbd_sender.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/drbd/drbd_sender.c b/drbd/drbd_sender.c
index 5d67612..e90135e 100644
--- a/drbd/drbd_sender.c
+++ b/drbd/drbd_sender.c
@@ -2013,7 +2013,10 @@ void drbd_rs_controller_reset(struct drbd_peer_device *peer_device)
 	atomic_set(&peer_device->device->rs_sect_ev, 0);  /* FIXME: ??? */
 	peer_device->rs_last_mk_req_kt = ktime_get();
 	peer_device->rs_in_flight = 0;
-	peer_device->rs_last_events = (int)part_stat_read_accum(disk->part0, sectors);
+
+/* TODO: WinDRBD: rs_last_events and part_stat_read_accum not implemented sync throtteling does not work. */
+
+        peer_device->rs_last_events = 0;
 
 	/* Updating the RCU protected object in place is necessary since
 	   this function gets called from atomic context.
-- 
2.17.0

